lib/src/voip/call_session.dart

```diff
@@ -946,6 +945,7 @@ class CallSession {
       return;
     }
     Logs().d('[VOIP] Rejecting call: $callId');
+    setCallState(CallState.kEnding);
```

Comment:
the sending of the reject event is in `sendCallReject` though, are you sure the commit message makes sense?

Reply:
yep, the actual problem was `setCallState` was called right before checking the state

Reply:
The if condition was always true because the state will be `CallState.kEnding` and only hangup code will be executed

Reply:
ah makes sense thanks!